source/FWU/MBFW/chapter2-uefi.md

```diff
@@ -2,22 +2,85 @@
 
 # UEFI spec
 
-
 ## UEFI Version
 
-This document uses version 2.9 of the UEFI specification [UEFI]_.
-
+This document uses version 2.11 of the UEFI specification [UEFI]_.
 
 ## UEFI Compliance
 
 All [UEFI]_ features required by [EBBR]_ are assumed.
 
-
 ## UEFI Extra Dependencies
 
 The capsules installed using the procedure defined in this document must be formatted according to the FMP defined format [UEFI]_ § 23.1 - Firmware Management Protocol.  
 The FMP instance must provide the `GetImageInfo` and `SetImage` functions to be used as the UpdateCapsule backend.
 
+There are two ways of accepting a new update or reverting an existing one.
+- If SetVariable at RunTime is supported `ABAction` UEFI variable is used.
+- If SetVariable at Runtime is not supported an empty capsule with a specific GUID is used.
+
+### Acceptance and Revert requests in firmware
+
+When UEFI receives an acceptance or downgrade request it must:
+
+1. Call the FWU primitive `fwu_set_active` [FWU]_ if the flash store is owned by the Secure World.
+2. Set the `active_index` field in the FWU Metadata [FWU]_ if the flash store is owned by the Normal World.
+3. Restore EFI variables to their previous state, if changed during the firmware upgrade.
+
+### Explicit Capsule Acceptance
+
+The firmware can skip the steps described in [OS Requested FW Acceptance](#os-requested-fw-acceptance) and immediately accept a new firmware.  
+In order to do so, the OS must set the `EFI_CAPSULE_HEADER.Flags[15]` to `1` during the capsule creation.  
+Other images may be implicitly accepted by the UEFI implementation.
+
+### Exchanging information for AB updates between the OS and the Firmware
+
+The firmware and an Operating System may exchange information through the `ABAction` and `ABStatus`
+UEFI variable as follows:
+
+-  The `ABStatus` variable returns a 64-bit unsigned integer owned by the firmware and indicates the
+status of the update. This variable is set by firmware, and cannot be modified by the OS.
+
```

Comment:
Nit: should we briefly describe the ABAction variable here?

Reason: the paragraph at line 38 implies ABAction and ABStatus will be described in the next lines.